### ISSUE TITLE
"Initialization" Output Returned as Stdout

### ISSUE THREAD

**Issue by BoweFlex** [minio/mc#5003]
The first time that any mc command is run, the following output is given:

```bash
mc: Configuration written to `/root/.mc/config.json`. Please update your access credentials.
mc: Successfully created `/root/.mc/share`.
mc: Initialized share uploads `/root/.mc/share/uploads.json` file.
mc: Initialized share downloads `/root/.mc/share/downloads.json` file.
```

Because of this, some situations cannot be automated properly due to this being returned when retrieving stdout from a command that just happens to be the first execution of mc (i.e. putting the output of an `mc cat` command into a variable. This is especially problematic with something like running an automated job pushing data through mc on a docker container with environment variables to control MC access.

## Expected behavior

I think there are a few potential fixes, but without looking more I'm not positive what makes the most sense for this project:
1. Add an optional --quiet flag to filter out some "informational" messages like this.
2. Add an `mc init` command, to create this configuration without any other expected output.
3. Change sections that output informational messages (i.e. https://github.com/minio/mc/blob/master/cmd/main.go#L271) to use console.Errorln rather than console.Infoln, which should route these messages to stderr rather than stdout and allow users to filter to receive only the output they want.

## Actual behavior

When run for the first time, or after removing config (i.e. `rm -rf /root/.mc`):
```bash
mc cat my_subdir/base_timestamp
mc: Configuration written to `/root/.mc/config.json`. Please update your access credentials.
mc: Successfully created `/root/.mc/share`.
mc: Initialized share uploads `/root/.mc/share/uploads.json` file.
mc: Initialized share downloads `/root/.mc/share/downloads.json` file.
<contents of file>
```
Results are the same with something like:
```bash
mc cat my_subdir/base_timestamp 2>/dev/null
mc: Configuration written to `/root/.mc/config.json`. Please update your access credentials.
mc: Successfully created `/root/.mc/share`.
mc: Initialized share uploads `/root/.mc/share/uploads.json` file.
mc: Initialized share downloads `/root/.mc/share/downloads.json` file.
<contents of file>
```

## Steps to reproduce the behavior

Start docker container w/ mc installed, and/or remove /root/.mc, and try running an mc command.

## mc --version
Confirmed on both:

- mc version RELEASE.2024-04-29T09-56-05Z
- mc version RELEASE.2021-03-23T05-46-11Z

## System information
Tested on rocky linux 9

**Comment by taran-p** [comment]
Does the `--quiet` flag fit your needs?

**Comment by BoweFlex** [comment]
> Does the `--quiet` flag fit your needs?

No, this output is returned either way. I didn't realize the flag was already included, it must not be working as expected.

Thanks @harshavardhana

**Comment by sivanantham-gnanavel** [comment]
@harshavardhana @BoweFlex I tried using the `--quiet` (`-q`) flag, and it skips printing the initialization details. Hope this resolves the issue.

### Standard Output
<img width="754" alt="Image" src="https://github.com/user-attachments/assets/0b507e93-a740-407d-85e8-c5a81539c3eb" />

### Quiet Output
<img width="409" alt="Image" src="https://github.com/user-attachments/assets/b7d65581-1931-40b9-8ab0-eb9fc521d7b4" />

**Comment by BoweFlex** [comment]
@sivanantham-gnanavel was this added recently? I could've sworn I tried using the `-q` flag before and it didn't work (at least I said I did in this discussion). However, I just tested this in a rocky container and you're correct that it hides that output when using the `-q` flag.

**Comment by sivanantham-gnanavel** [comment]
@BoweFlex I tested this on the same versions you mentioned, and the `--quiet` (`-q`) flag is working as expected. I’m not sure what might have caused the issue earlier, but since the version hasn’t changed, the behaviour should remain the same.

<img width="364" alt="Image" src="https://github.com/user-attachments/assets/da301d8e-20ac-4f66-960d-357fbf3922d3" />

<img width="361" alt="Image" src="https://github.com/user-attachments/assets/637b5f3b-3bfd-4861-812c-a94bf0b6e163" />
